public/app/screens/main/navigation/navigation.jsx

```diff
@@ -1,21 +1,30 @@
 var React = require('react'),
-  Link = require('react-router').Link,
+  cx = require('react/lib/cx'),
+  Router = require('react-router'),
+  State = Router.State,
+  Link = Router.Link,
   styleMixin = require('mixins/style-mixin');
 
 var Navigation = React.createClass({
-  mixins: [ styleMixin(require('./style.scss')) ],
+  mixins: [styleMixin(require('./style.scss')), State],
   render: function render () {
     // TODO: Use mixins instead of CSS classes
     // TODO: Pass number of elements to mixin instead of specifying class name
     var classes = 'cc_navigation icon-bar two-up ' + this.props.orientation;
 
     var items = this.props.items.map(function mapper (item, i) {
+      var classNames = cx({
+        'cc_navigation__link': true,
+        'item': true,
+        // also mark link as active, if subpath of link is opened
+        'active': this.getPath().indexOf('/' + item.linkTo) === 0
```

Comment:
`active` is a CSS class provided by foundation. It will style the navigation item as selected/active. The comment thus says: `also mark link as active, if subpath of link is opened`.